lablqml/src_tests/test3.qml

```diff
@@ -0,0 +1,45 @@
+import QtQuick 2.1
+import QtQuick.Controls 1.0
+
+/* Demonstraction of direct property binding
+OCaml binds to objects named "test" and "mirror".
+When test changes, the binding function is called and the
+mirror's text value is set */
+
+ApplicationWindow {
+    visible: true
+    QtObject {
+	id: t
+	objectName: "test"
+	property string msg: "Hello world! Hello Qml with OCaml!"
+
+	// QtObjects can only be nested as properties
```

Comment:
> You will notice that child objects can be added to any Item-based type without explicitly adding them to the children property. This is because the default property of Item is its data property, and any items added to this list for an Item are automatically added to its list of children.

from: http://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html#default-properties